libs/twitch/twitch.go

```diff
@@ -68,8 +68,8 @@ func NewAppClientWithContext(ctx context.Context, config cfg.Config, tokensGrpc
 	return client, nil
 }
 
-func NewUserClient(userID string, config cfg.Config, tokensGrpc tokens.TokensClient) (*helix.Client, error) {
-	return NewUserClientWithContext(context.Background(), userID, config, tokensGrpc)
+func NewUserClient(ctx context.Context, userID string, config cfg.Config, tokensGrpc tokens.TokensClient) (*helix.Client, error) {
```

Comment:
There is `NewUserCLientWithContext`, revert changes in this file please.

Reply:
fixed